docs/development_suite/deploy/configure-github-actions.md

```diff
@@ -0,0 +1,126 @@
+
+To enable your project to be deployed using GitHub Actions, you should edit the configuration of the project through the Console CMS 
+(if you do not have access to it, ask your instance administrator). In particular, the `environments.deploy` object 
+of the configuration should contain the following information:
+
+```json
+{
+  "deploy": {
+    "type": "github-actions",
+    "providerId": "your-github-provider-id"
+  }
+} 
+```
+
+:::info
+if you haven't set up a GitHub provider yet, [see this page](/development_suite/set-up-infrastructure/configure-provider.mdx) to learn how to configure it.
+Remember that the Access Token configured to your provider needs at least `read-level access scope for the "Deployments" of the organization.
+In order for GitHub to send webhook payloads, your server needs to be accessible from the Internet.
+:::
+
+:::info
+if you haven't set up an environment yet, [see this page](/development_suite/set-up-infrastructure/add-environment.mdx) to learn how to configure it.
+:::
+
+## Workflow
+
+In your GitHub repository, create a new file `<root>/.github/workflows/deploy.yml`.
+
+Here is an example that you can use to start:
+```yaml
+name: Deploy
+
+on: deployment
+
+jobs:
+  deploy-ENVIRONMENT_NAME:
+    name: Deploy to ENVIRONMENT_NAME environment
+    if: ${{ github.event.deployment.environment }} == ENVIRONMENT_NAME
+    concurrency: ENVIRONMENT_NAME
+    permissions:
+      deployments: write
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Update deployment status (pending)
+        if: success()
+        uses: chrnorm/deployment-status@v2
+        with:
+          token: '${{ github.token }}'
+          state: 'pending'
+          deployment-id: ${{ github.event.deployment.id }}
+
+      - name: Deploy my app
+        run: |
+          # add your deployment code here
+          echo Deploying...
+          sleep 10s
+          echo Done!
+
+      - name: Update deployment status (success)
+        if: success()
+        uses: chrnorm/deployment-status@v2
+        with:
+          token: '${{ github.token }}'
+          state: 'success'
+          deployment-id: ${{ github.event.deployment.id }}
+
+      - name: Update deployment status (failure)
+        if: failure()
+        uses: chrnorm/deployment-status@v2
+        with:
+          token: '${{ github.token }}'
+          state: 'failure'
+          deployment-id: ${{ github.event.deployment.id }}
+```
+
+When users deploy for the first time, the Console release automatically creates the environments configured inside the project, 
+otherwise, you can [follow this tutorial](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#creating-an-environment) 
+to set up your environments.
+
+With the above configuration, when a user presses the `Deploy` button inside the Console, the following steps will be executed:
```

Comment:
```suggestion
With the above configuration, when a user clicks the `Deploy` button inside the Console, the following steps will be executed:
```

Reply:
**Generic guideline**
Use "Press" when referring to a hardware button and "Click" when referring to an item in the UI